package.json

```diff
@@ -21,7 +21,7 @@
     "bootstrap-switch": "3.3.2",
     "colors": "1.1.2",
     "css-loader": "^0.26.1",
-    "eonasdan-bootstrap-datetimepicker": "~4.15",
+    "eonasdan-bootstrap-datetimepicker": "~4.17",
```

Comment:
Nevermind, just noticed your commit message further down. We'll need to fix the test.